fast64_internal/utility.py

```diff
@@ -157,18 +157,30 @@ def checkObjectReference(obj, title):
         )
 
 
-def selectSingleObject(obj: bpy.types.Object):
-    bpy.ops.object.select_all(action="DESELECT")
+def getActiveObject():
+    return bpy.context.view_layer.objects.active
+
+
+def setActiveObject(obj: bpy.types.Object):
     obj.select_set(True)
     bpy.context.view_layer.objects.active = obj
 
 
-def parentObject(parent, child):
-    bpy.ops.object.select_all(action="DESELECT")
+def deselectAllObjects():
+    for obj in bpy.data.objects:
+        obj.select_set(False)
+
+
+def selectSingleObject(obj: bpy.types.Object):
+    deselectAllObjects()
+    setActiveObject(obj)
+
+
+def parentObject(parent: bpy.types.Object, child: bpy.types.Object):
+    deselectAllObjects()
 
     child.select_set(True)
-    parent.select_set(True)
-    bpy.context.view_layer.objects.active = parent
+    setActiveObject(parent)
     bpy.ops.object.parent_set(type="OBJECT", keep_transform=True)
```

Comment:
what is this doing exactly? setting a parent object? if so then why not naming this ``setParentObject`` to avoid any confusion?

Reply:
yeah it's setting `child`'s parent to `parent`

the function name is to be understood as "parent the object" (the "to parent" verb)

but yeah I can change it to `setParentObject(child, parent)` it will be easier to understand

Reply:
yea with the current name you could think it's doing something to an object's parent

Reply:
<img src="https://cdn.discordapp.com/emojis/1020465177776373760.webp?size=240&quality=lossless" alt="flink" height="24px">